### DISCUSSION TITLE
How can I connect filter expressions with logic OR in OAL?

### DISCUSSION THREAD

**Discussion by karlio66** [apache/skywalking#11533] [Q&A]
I want to change service_sla like：service_sla = from(Service.*).percent(status == true OR httpResponseStatusCode < 500);
but the syntax is wrong，office docs only introduce logic AND, I can't find logic OR.

**Comment by wu-sheng** [comment]
We don't support that. And `status == true OR httpResponseStatusCode < 500` conditions don't make sense.
